MeshLib/MeshGenerators/VtkMeshConverter.cpp

```diff
@@ -455,6 +456,12 @@ void VtkMeshConverter::convertArray(vtkDataArray &array, MeshLib::Properties &pr
 		return;
 	}
 
+	if (vtkUnsignedLongArray::SafeDownCast(&array))
+	{
+		VtkMeshConverter::convertTypedArray<std::size_t>(array, properties, type);
```

Comment:
The assumption that `std::size_t` is equal to `unsigned long` is **not** always fulfilled. As far as I know the C++ standard does not specify the size of `std::size_t` nor does it specify the size of `unsigned long`. Is an vtk unsigned long garantied to be equal to a compiler and architecture dependend `unsigned long`? What is the correct vtk type for storing `std::size_t`?